contracts/cw1-whitelist/src/contract.rs

```diff
@@ -0,0 +1,198 @@
+use anyhow::Error;
+use cosmwasm_std::{from_slice, Addr, Binary, Deps, DepsMut, Empty, Env, MessageInfo, Response};
+use cw1::msg::{ExecMsg, QueryMsg};
+use cw1::Cw1;
+use cw1::*;
+use cw_storage_plus::Map;
+use sylvia::contract;
+
+use self::contract::InstantiateMsg;
+
+pub struct Cw1WhitelistContract {
+    members: Map<'static, Addr, Empty>,
```

Comment:
I think we want this to be public - if another contract would like to extend this one it would probably want to have access to its state. However it decreases hermetization, and it's not clear how to approach this, we will build some proper patterns in future so we can leave it like it is, for now, will see what works best when we will add more contracts.